src/main/java/com/gw/exercises/task1/helloName.java

```diff
@@ -0,0 +1,33 @@
+package com.gw.exercises.task1;
+
+public class helloName {
```

Comment:
The name of the class should start with the capital letter.

Reply:
helloName  - > HelloName